paper/paper.md

```diff
@@ -0,0 +1,41 @@
+# CMakePPLang
+
+## Summary
+
+
+
+## Statement of Need
+
+CMake is small, extensible, and exceeds at generating build systems for many combinations of platforms, compilers, and build configurations.
+
+CMake has become the *de facto* standard tool for building C, C++, and Fortran
+programs of moderate to large size. As the size of a project increases, the
+complexity of the CMake code to build it tends to also increase. Object-
+oriented programming excels at managing large, complex code bases, and there
+is an increasing need for this in the CMake language.
+
+As more tooling is built around CMake to make building projects with CMake
+easier and less error prone, these tools will need to be designed in a
+maintainable and testable way. Object-oriented programming excels at creating
```

Comment:
Again a reference would bolster this claim (could quite possibly be the same reference as above).

Reply:
I can probably check one of my old software engineering books for this reference and the one above.